suite-native/intl/src/en.ts

```diff
@@ -1005,6 +1005,23 @@ export const en = {
                 standard: 'Standard wallet',
                 defaultPassphrase: 'Passphrase wallet #{index}',
             },
+            autoEject: {
+                switch: {
+                    title: 'Auto eject',
+                    description: 'Eject all wallets automatically after unplugging Trezor',
+                    alert: {
+                        titleNoConnectedTrezor: 'Enabling auto eject will eject all wallets',
+                        titleConnectedTrezor: ' after disconnecting Trezor',
+                        description:
+                            'You will need to reconnect your wallets to see your balances again.',
+                        primaryButtonTitle: 'Enable auto eject',
```

Comment:
according design autoeject, should be together, actually why is there `auto-eject `and `autoeject`? :D https://www.figma.com/design/aybSBSnUWPebHVCpFvVtrj/Auto-eject?node-id=2204-4956&t=5OZyuuXRg5fxen1Z-1

Reply:
This is what it looked like on 18th jul 😆 I think I would just merge and see if its a problem in design QA.
<img width="420" height="433" alt="image" src="https://github.com/user-attachments/assets/b36a22eb-4658-4f50-aec8-71a6ccc4c726" />